src/DataValues/TimeValue.php

```diff
@@ -31,15 +31,23 @@ class TimeValue extends DataValueObject {
 	const PRECISION_SECOND = 14;
 
 	/**
-	 * Point in time, represented per ISO8601, in the format +0000000000002013-01-01T00:00:00Z.
-	 * The year must be signed. It may be padded with zero characters to have up to 16 digits.
+	 * Timestamp describing a point in time. The actual format depends on the calendar model.
+	 *
+	 * Gregorian and Julian dates use the same YMD ordered format, resembling ISO 8601, e.g.
+	 * +2013-01-01T00:00:00Z. In this format the year is always signed and padded with zero
+	 * characters to have between 4 and 16 digits. Month and day can be zero, indicating they are
+	 * unknown. The timezone suffix Z is meaningless and must be ignored. Use $timezone instead.
```

Comment:
This discussion should not happen in this patch. Please open a ticket if you think this should change. However, I disagree. As @Jc3s5h pointed out the ISO norm does this by truncating the timestamp string. We do this with our "precision" field. Everything behind is meaningless and must be ignored. In the end it doesn't matter if month and day are "01", "00" or "**". I'm voting for "00".